rs/index_server/src/main.rs

```diff
@@ -126,6 +127,15 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         flush_worker_threads.push(collection_manager_flush_thread);
     }
 
+    // Start the metrics server
+    let metrics_addr = SocketAddr::new(addr.ip(), addr.port() + 1);
```

Comment:
let's call this HTTP server. metrics server is only restricted to `/metrics` This server might be used for other purposes later (for example, if customers don't want to send gRPC and prefer HTTP instead)